src/wechaty_puppet_itchat/puppet.py

```diff
@@ -1098,10 +1043,9 @@ async def on_message_group_text(msg):
             )
 
             self._event_stream.emit('message', event_message_payload)
-            await asyncio.sleep(0.1)
 
         @self.itchat.msg_register([PICTURE, RECORDING, ATTACHMENT, VIDEO])
-        async def on_message_person_file(msg):
+        def on_message_person_file(msg):
```

Comment:
The register func should be async. You can refactor the decorator of `msg_register ` to run async and sync functions.

Reply:
Done~